profiles/default.yml.example

```diff
@@ -10,6 +10,9 @@ connection:
 kanboard:
   project_id: 1
 
+configuration:
```

Comment:
is this a new, global configuration? shouldn't it be at least under kanboard above?

Reply:
oh I get it now, there's not example of task specific configuration, should we perhaps nest it under global key?

Reply:
anyway, since you and I are the only users so we could fix it later... I'll be playing with the PR in my instance and if it works, I'll merge